config/jobs/kubernetes-sigs/apiserver-network-proxy/apiserver-network-proxy-presubmits.yaml

```
@@ -0,0 +1,17 @@
+presubmits:
+  kubernetes-sigs/apiserver-network-proxy:
+  - name: pull-apiserver-network-proxy-test
+    always_run: true
+    skip_report: false
+    decorate: true
+    spec:
+      containers:
+      - image: golang:1.12
+        command:
+        - make
+        args:
+        - test
+    annotations:
+      testgrid-dashboards: sig-cloudprovider-apiserver-network-proxy
```

```suggestion
      testgrid-dashboards: sig-cloud-provider-apiserver-network-proxy
```

To match https://github.com/kubernetes/test-infra/pull/15011#pullrequestreview-309663164.